.github/ISSUE_TEMPLATE/new-patch-release.md

```diff
@@ -19,8 +19,11 @@ Please do not remove items from the checklist
   - [ ] Upload release artefacts generated by `prepare-release.sh` script above to the draft release
 - [ ] An OWNER runs
      `git tag -s -m "NFD release $VERSION" $VERSION`
+- [ ] An OWNER runs
+     `git tag -s -m "NFD API release $VERSION" api/nfd/$VERSION`     
 - [ ] An OWNER pushes the tag with
       `git push $VERSION`
+      `git push api/nfd/$VERSION`
```

Comment:
push both tags with a single command

```suggestion
      `git push $VERSION api/nfd/$VERSION`
```

Alternatively have two separate checklist items like for the tagging above. WDYT?

Reply:
Done